src/components/03-components/01-actions/actions.njk

```diff
@@ -7,6 +7,11 @@
       <span class="sr-only">Screen reader description</span>
     </button>
 
+    <button class="sds-button--actions">
```

Comment:
This is resulting in the existing in a button with the old action icon in it, placed next to the new action icon. If the old one is wrong, why not replace it with this new one?

Reply:
I think Dave wants to keep both looks (blue and white).